src/test/debugger/setup.test.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+/* This setup is just to install the source-map-support, so we'll get TypeScript stack traces
+   for the tests */
+import * as fs from "fs";
+import * as path from "path";
+
+// source-map-support has a bug that crashes some of our tests. We fix it
```

Comment:
They are aware, but I'm not sure when or if this is going to get fixed: https://github.com/evanw/node-source-map-support/issues/131